### ISSUE TITLE
Detect version mismatches between the plugin and the adobe_xd package.

### ISSUE THREAD

**Issue by gskinner** [AdobeXD/xd-to-flutter-plugin#80]
It would help prevent issues like #74 if we could detect and surface mismatches between the plugin versions and the adobe_xd package version it relies on.

A stretch goal would be to fix it, though we've avoided writing to pubspec.yaml thus far.

**Comment by Dilshan1997** [comment]
Who use the flutter_svg package and xd package use this version's dependencies.
```
environment:
  sdk: ">=2.10.4 <3.0.0"

dependencies:
  flutter:
    sdk: flutter
  flutter_svg: ^0.17.0+1
  adobe_xd: ^1.0.0+1

```

**Comment by AliYar-Khan** [comment]
> 
> 
> Who use the flutter_svg package and xd package use this version's dependencies.
> 
> ```
> environment:
>   sdk: ">=2.10.4 <3.0.0"
> 
> dependencies:
>   flutter:
>     sdk: flutter
>   flutter_svg: ^0.17.0+1
>   adobe_xd: ^1.0.0+1
> ```

This works for me ...

**Comment by gskinner** [comment]
This is largely resolved in v2.0.0. Closing in favor of the more general issue #26
